CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### 🛑 Breaking changes 🛑
+
+- (Splunk) `httpforwarder`: Remove extension named `httpforwarder`, use `httpforwarderextension` instead.
```

Comment:
```suggestion
- (Splunk) `httpforwarder`: Remove extension named `httpforwarder`, use `httpforwarderextension` instead. ([#4455](https://github.com/signalfx/splunk-otel-collector/pull/4455))
```

Reply:
the type of the extension is the same and the change will not affect the end users. Config YAML will not change.

Reply:
In that case, I'll remove the changelog 👍